paasta_tools/marathon_tools.py

```diff
@@ -771,6 +771,10 @@ def get_marathon_services_running_here_for_nerve(cluster, soa_dir):
                 nerve_list.append((registration, nerve_dict))
         except KeyError:
             continue  # SOA configs got deleted for this app, it'll get cleaned up
+        except NoConfigurationForServiceError:
```

Comment:
Go ahead and throw this on line 772 along with the key error, no need to log or anything.